PORT-13363-bug-it-is-not-mentioned-in-the-docs-that-we-support-gitops-in-self-hosted-bitbucket (#2104)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/bitbucket/bitbucket.md

Port's Bitbucket integration allows you to model Bitbucket resources in your software catalog and ingest data into them.

> :::info Using Bitbucket Server (Self-Hosted)?

Reviewer: ```suggestion
:::info Bitbucket Server (Self-Hosted)
```

> This documentation covers Port's integration with **Bitbucket Cloud**. If you're looking for information about Bitbucket Server (Self-Hosted), please refer to our [Bitbucket Server integration documentation](/build-your-software-catalog/custom-integration/webhook/examples/bitbucket-server/bitbucket-server.md).

Reviewer: ```suggestion
This documentation covers Port's integration with **Bitbucket Cloud**. For information about integrating with Bitbucket Server (Self-Hosted), please refer to the [Bitbucket Server integration documentation](/build-your-software-catalog/custom-integration/webhook/examples/bitbucket-server/bitbucket-server.md).
```
